src/manager.wsgi

```diff
@@ -948,6 +948,13 @@ def application(environ, start_response):
 
     custom_url = "%s/__custom__" % match.group(1)
 
+    ftp_box = ""
+    if CONFIG["UseFTPTasks"]:
+        with open("/usr/share/retrace-server/manager_ftp_box.xhtml") as file:
+            # Read 1 MiB
+            ftp_box = file.read(1 << 20)
```

Comment:
I think I'd prefer the HTML fragment to stay in this file in some variable/constant. It seems a bit excessive to create a special file for such a small chunk.

Reply:
Great feedback - fixed in 3a31b86.